app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java

```diff
@@ -1,86 +1,81 @@
 package ju.video.player.view.videoconvert;
 
-import javax.swing.BorderFactory;
 import javax.swing.JPanel;
-import javax.swing.border.Border;
 
 import ju.video.player.controller.ConvertController;
-import ju.video.player.view.MainFrame;
 import ju.video.player.view.commons.Colors;
-import ju.video.player.view.information.search.FolderChooserButton;
-import ju.video.player.view.materialDesing.ResponsiveSwingMaterialDesign;
-import ju.video.player.view.materialDesing.components.date.fields.DateField;
-import ju.video.player.view.materialDesing.components.image.ImageLabel;
-import ju.video.player.view.materialDesing.components.utils.ImageRoundedBorder;
 import ju.video.player.view.materialDesing.components.utils.RoundedBorder;
 import ju.video.player.view.materialDesing.display.FrameUtility;
-import ju.video.player.view.materialDesing.layouts.ResponsiveLayout;
-import java.awt.event.ActionListener;
-import java.awt.Dimension;
 
 import javax.swing.JButton;
-
 import javax.swing.*;
 import java.awt.*;
```

Comment:
Please specify the Javax Swing and Java awt libraries used.

Reply:
The libraries have already been changed to the specific ones.